models/decision.go

```diff
@@ -52,6 +52,17 @@ type DecisionWithRuleExecutions struct {
 	RuleExecutions []RuleExecution
 }
 
+type DecisionsByVersionByOutcome struct {
```

Comment:
see the remark on the repository: this is actually a regression, the initial model (and dto) with version,outcome,score,total count was the correct one - and the results for both versions of the scenario should be unioned rather than joined.